pkg/provider/github/parse_payload.go

```diff
@@ -370,3 +375,38 @@ func (v *Provider) handleIssueCommentEvent(ctx context.Context, event *github.Is
 	v.Logger.Infof("issue_comment: pipelinerun %s on %s/%s#%d has been requested", action, runevent.Organization, runevent.Repository, runevent.PullRequestNumber)
 	return v.getPullRequest(ctx, runevent)
 }
+
+func (v *Provider) handleCommitCommentEvent(ctx context.Context, event *github.CommitCommentEvent) (*info.Event, error) {
+	action := "recheck"
```

Comment:
maybe that action name meaning is too broad? can you call it something more specific?

Reply:
used `push`